### PR TITLE
Issue7414 Inconsistent pair convention for Cutout2D position and size

### LINKED ISSUE #7414: Cutout2D: Inconsistent array convention (needs documentation)
The ordered pair convention for Cutout2D is inconsistent and somewhat surprising.  While the `position` argument is specified as `(x, y)`, the `size` argument is reversed.  From the documentation:

> If `size` has two elements, they should be in `(ny, nx)` order.

  While the `(y, x)` order is standard convention for most numpy/scipy functions, it is not intuitive to specify the position and size in reverse order from each other within the same function call.

### PR DESCRIPTION
**Pull request by SG004**
EDIT: Fix #7414

### CODE REVIEW

File: docs/nddata/utils.rst [hunk 1]
```diff
@@ -26,6 +26,8 @@ postage stamp cutout image from a 2D array.  If an optional
 
 First, let's simulate a single source on a 2D data array. If you would like to
 simulate many sources, see :ref:`bounding-boxes`.
+.. note:: The pair convention is inconsistent. The position argument is specified
+.. as (x,y), however the size argument is reverse (y,x)
```
Comment (pllim) [inline docs/nddata/utils.rst]: You would need to fix the RST formatting. I'll leave subpackage maintainer to review the actual content.

**Comment by bsipocz** [comment]
This doesn't renders properly in the docs: https://10646-2081289-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/nddata/utils.html?highlight=inconsistent

Also, maybe a better place would be for this in the docstring rather than in the narrative docs, what do others think?

**Comment by astrofrog** [comment]
i think we want to be careful to not call this inconsistent but instead to just say note that the order is different. It's actually quite handy to just be able to pass a numpy shape to ``size``.

**Comment by SG004** [comment]
I will make the changes

**Comment by mwcraig** [comment]
@SG004 -- thanks for this contribution! We really appreciate when folks notice these things and take the time to fix them. I agree with @astrofrog that you should avoid saying it is inconsistent. Maybe different is better word.

Also, along the lines of what @bsipocz suggested can you please also check to see if this change makes sense in the docstring for `Cutout2D` (see https://github.com/astropy/astropy/blob/master/astropy/nddata/utils.py#L476)?

**Comment by SG004** [comment]
@mwcraig I've changed the wording of the note.

As for if it makes sense in the docstring. It adds a bit of clarity, and makes sense.

**Pull request closed without merging**